Source: Local
Upstream: no
Reason: This patch is used to delete all references to remote files for building
--
--- a/subprojects/docs/docs.gradle
+++ b/subprojects/docs/docs.gradle
@@ -224,6 +224,8 @@
 
 def javaApiUrl = "http://docs.oracle.com/javase/1.5.0/docs/api/"
 def groovyApiUrl = "http://groovy.codehaus.org/gapi/"
+def javaApiLocalUrl = localFiles + "/javaApi/"
+def groovyApiLocalUrl = localFiles + "/groovyApi/"
 
 task javadoc(type: Javadoc) {
     ext.stylesheetFile = file("src/docs/css/javadoc.css")
@@ -255,7 +257,7 @@
 task configureGroovydoc {
     doFirst {
         project.configure(groovydoc) {
-            [javaApiUrl, groovyApiUrl].each {
+            [javaApiLocalUrl, groovyApiLocalUrl].each {
                 link(it, *(new URL("$it/package-list").text.tokenize("\n")))
             }
             docTitle = "Gradle API $version"--
--- a/subprojects/internal-integ-testing/internal-integ-testing.gradle
+++ b/subprojects/internal-integ-testing/internal-integ-testing.gradle
@@ -38,7 +38,7 @@
         def url = "http://services.gradle.org/versions/all"
         logger.info "Getting the released versions from: $url"
 
-        def json = new URL("http://services.gradle.org/versions/all").text
+        def json = new URL(localFiles + "/versions").text
         def destFile = new File(ext.destDir, "all-released-versions.json")
         assert destDir.mkdirs() || destDir.exists()
         destFile.text = json
--- /dev/null
+++ b/tempfiles/versions
@@ -0,0 +1,181 @@
+[{
+  "version":"1.1-rc-1-20120426000048+0200",
+  "buildTime":"20120426000048+0200",
+  "current":false,
+  "snapshot":true,
+  "nightly":true,
+  "rcFor":null,
+  "broken":false,
+  "downloadUrl":"http:\/\/services.gradle.org\/distributions-snapshots\/gradle-1.1-rc-1-20120426000048+0200-bin.zip"
+},{
+  "version":"1.0-rc-1",
+  "buildTime":"20120411121324+0100",
+  "current":false,
+  "snapshot":false,
+  "nightly":false,
+  "rcFor":null,
+  "broken":false,
+  "downloadUrl":"http:\/\/services.gradle.org\/distributions\/gradle-1.0-rc-1-bin.zip"
+},{
+  "version":"1.0-milestone-9",
+  "buildTime":"20120313171009+0100",
+  "current":false,
+  "snapshot":false,
+  "nightly":false,
+  "rcFor":null,
+  "broken":false,
+  "downloadUrl":"http:\/\/services.gradle.org\/distributions\/gradle-1.0-milestone-9-bin.zip"
+},{
+  "version":"1.0-milestone-8a",
+  "buildTime":"20120220185357+0100",
+  "current":false,
+  "snapshot":false,
+  "nightly":false,
+  "rcFor":null,
+  "broken":false,
+  "downloadUrl":"http:\/\/services.gradle.org\/distributions\/gradle-1.0-milestone-8a-bin.zip"
+},{
+  "version":"1.0-milestone-8",
+  "buildTime":"20120214022451+0100",
+  "current":false,
+  "snapshot":false,
+  "nightly":false,
+  "rcFor":null,
+  "broken":false,
+  "downloadUrl":"http:\/\/services.gradle.org\/distributions\/gradle-1.0-milestone-8-bin.zip"
+},{
+  "version":"1.0-milestone-7",
+  "buildTime":"20120105102443+0000",
+  "current":false,
+  "snapshot":false,
+  "nightly":false,
+  "rcFor":null,
+  "broken":false,
+  "downloadUrl":"http:\/\/services.gradle.org\/distributions\/gradle-1.0-milestone-7-bin.zip"
+},{
+  "version":"1.0-milestone-6",
+  "buildTime":"20111117065412+0100",
+  "current":false,
+  "snapshot":false,
+  "nightly":false,
+  "rcFor":null,
+  "broken":false,
+  "downloadUrl":"http:\/\/services.gradle.org\/distributions\/gradle-1.0-milestone-6-bin.zip"
+},{
+  "version":"1.0-milestone-5",
+  "buildTime":"20111025055608+0200",
+  "current":false,
+  "snapshot":false,
+  "nightly":false,
+  "rcFor":null,
+  "broken":false,
+  "downloadUrl":"http:\/\/services.gradle.org\/distributions\/gradle-1.0-milestone-5-bin.zip"
+},{
+  "version":"1.0-milestone-4",
+  "buildTime":"20110728103822+0200",
+  "current":false,
+  "snapshot":false,
+  "nightly":false,
+  "rcFor":null,
+  "broken":true,
+  "downloadUrl":"http:\/\/services.gradle.org\/distributions\/gradle-1.0-milestone-4-bin.zip"
+},{
+  "version":"1.0-milestone-3",
+  "buildTime":"20110425174011+1000",
+  "current":false,
+  "snapshot":false,
+  "nightly":false,
+  "rcFor":null,
+  "broken":false,
+  "downloadUrl":"http:\/\/services.gradle.org\/distributions\/gradle-1.0-milestone-3-bin.zip"
+},{
+  "version":"1.0-milestone-2",
+  "buildTime":"20110407163255+1000",
+  "current":false,
+  "snapshot":false,
+  "nightly":false,
+  "rcFor":null,
+  "broken":false,
+  "downloadUrl":"http:\/\/services.gradle.org\/distributions\/gradle-1.0-milestone-2-bin.zip"
+},{
+  "version":"1.0-milestone-1",
+  "buildTime":"20110227141320+1100",
+  "current":false,
+  "snapshot":false,
+  "nightly":false,
+  "rcFor":null,
+  "broken":false,
+  "downloadUrl":"http:\/\/services.gradle.org\/distributions\/gradle-1.0-milestone-1-bin.zip"
+},{
+  "version":"0.9.2",
+  "buildTime":"20110123133421+1100",
+  "current":false,
+  "snapshot":false,
+  "nightly":false,
+  "rcFor":null,
+  "broken":false,
+  "downloadUrl":"http:\/\/services.gradle.org\/distributions\/gradle-0.9.2-bin.zip"
+},{
+  "version":"0.9.1",
+  "buildTime":"20110102114057+1100",
+  "current":false,
+  "snapshot":false,
+  "nightly":false,
+  "rcFor":null,
+  "broken":false,
+  "downloadUrl":"http:\/\/services.gradle.org\/distributions\/gradle-0.9.1-bin.zip"
+},{
+  "version":"0.9",
+  "buildTime":"20101219125006+1100",
+  "current":false,
+  "snapshot":false,
+  "nightly":false,
+  "rcFor":null,
+  "broken":false,
+  "downloadUrl":"http:\/\/services.gradle.org\/distributions\/gradle-0.9-bin.zip"
+},{
+  "version":"0.9-rc-3",
+  "buildTime":"20101120131750+1100",
+  "current":false,
+  "snapshot":false,
+  "nightly":false,
+  "rcFor":null,
+  "broken":false,
+  "downloadUrl":"http:\/\/services.gradle.org\/distributions\/gradle-0.9-rc-3-bin.zip"
+},{
+  "version":"0.9-rc-2",
+  "buildTime":"20101027082405+1100",
+  "current":false,
+  "snapshot":false,
+  "nightly":false,
+  "rcFor":null,
+  "broken":false,
+  "downloadUrl":"http:\/\/services.gradle.org\/distributions\/gradle-0.9-rc-2-bin.zip"
+},{
+  "version":"0.9-rc-1",
+  "buildTime":"20100804080433+1100",
+  "current":false,
+  "snapshot":false,
+  "nightly":false,
+  "rcFor":null,
+  "broken":false,
+  "downloadUrl":"http:\/\/services.gradle.org\/distributions\/gradle-0.9-rc-1-bin.zip"
+},{
+  "version":"0.8",
+  "buildTime":"20090928140159+0200",
+  "current":false,
+  "snapshot":false,
+  "nightly":false,
+  "rcFor":null,
+  "broken":false,
+  "downloadUrl":"http:\/\/services.gradle.org\/distributions\/gradle-0.8-bin.zip"
+},{
+  "version":"0.7",
+  "buildTime":"20090720085013+0200",
+  "current":false,
+  "snapshot":false,
+  "nightly":false,
+  "rcFor":null,
+  "broken":false,
+  "downloadUrl":"http:\/\/services.gradle.org\/distributions\/gradle-0.7-bin.zip"
+}]
--- /dev/null
+++ b/tempfiles/javaApi/package-list
@@ -0,0 +1,166 @@
+java.applet
+java.awt
+java.awt.color
+java.awt.datatransfer
+java.awt.dnd
+java.awt.event
+java.awt.font
+java.awt.geom
+java.awt.im
+java.awt.im.spi
+java.awt.image
+java.awt.image.renderable
+java.awt.print
+java.beans
+java.beans.beancontext
+java.io
+java.lang
+java.lang.annotation
+java.lang.instrument
+java.lang.management
+java.lang.ref
+java.lang.reflect
+java.math
+java.net
+java.nio
+java.nio.channels
+java.nio.channels.spi
+java.nio.charset
+java.nio.charset.spi
+java.rmi
+java.rmi.activation
+java.rmi.dgc
+java.rmi.registry
+java.rmi.server
+java.security
+java.security.acl
+java.security.cert
+java.security.interfaces
+java.security.spec
+java.sql
+java.text
+java.util
+java.util.concurrent
+java.util.concurrent.atomic
+java.util.concurrent.locks
+java.util.jar
+java.util.logging
+java.util.prefs
+java.util.regex
+java.util.zip
+javax.accessibility
+javax.activity
+javax.crypto
+javax.crypto.interfaces
+javax.crypto.spec
+javax.imageio
+javax.imageio.event
+javax.imageio.metadata
+javax.imageio.plugins.bmp
+javax.imageio.plugins.jpeg
+javax.imageio.spi
+javax.imageio.stream
+javax.management
+javax.management.loading
+javax.management.modelmbean
+javax.management.monitor
+javax.management.openmbean
+javax.management.relation
+javax.management.remote
+javax.management.remote.rmi
+javax.management.timer
+javax.naming
+javax.naming.directory
+javax.naming.event
+javax.naming.ldap
+javax.naming.spi
+javax.net
+javax.net.ssl
+javax.print
+javax.print.attribute
+javax.print.attribute.standard
+javax.print.event
+javax.rmi
+javax.rmi.CORBA
+javax.rmi.ssl
+javax.security.auth
+javax.security.auth.callback
+javax.security.auth.kerberos
+javax.security.auth.login
+javax.security.auth.spi
+javax.security.auth.x500
+javax.security.cert
+javax.security.sasl
+javax.sound.midi
+javax.sound.midi.spi
+javax.sound.sampled
+javax.sound.sampled.spi
+javax.sql
+javax.sql.rowset
+javax.sql.rowset.serial
+javax.sql.rowset.spi
+javax.swing
+javax.swing.border
+javax.swing.colorchooser
+javax.swing.event
+javax.swing.filechooser
+javax.swing.plaf
+javax.swing.plaf.basic
+javax.swing.plaf.metal
+javax.swing.plaf.multi
+javax.swing.plaf.synth
+javax.swing.table
+javax.swing.text
+javax.swing.text.html
+javax.swing.text.html.parser
+javax.swing.text.rtf
+javax.swing.tree
+javax.swing.undo
+javax.transaction
+javax.transaction.xa
+javax.xml
+javax.xml.datatype
+javax.xml.namespace
+javax.xml.parsers
+javax.xml.transform
+javax.xml.transform.dom
+javax.xml.transform.sax
+javax.xml.transform.stream
+javax.xml.validation
+javax.xml.xpath
+org.ietf.jgss
+org.omg.CORBA
+org.omg.CORBA.DynAnyPackage
+org.omg.CORBA.ORBPackage
+org.omg.CORBA.TypeCodePackage
+org.omg.CORBA.portable
+org.omg.CORBA_2_3
+org.omg.CORBA_2_3.portable
+org.omg.CosNaming
+org.omg.CosNaming.NamingContextExtPackage
+org.omg.CosNaming.NamingContextPackage
+org.omg.Dynamic
+org.omg.DynamicAny
+org.omg.DynamicAny.DynAnyFactoryPackage
+org.omg.DynamicAny.DynAnyPackage
+org.omg.IOP
+org.omg.IOP.CodecFactoryPackage
+org.omg.IOP.CodecPackage
+org.omg.Messaging
+org.omg.PortableInterceptor
+org.omg.PortableInterceptor.ORBInitInfoPackage
+org.omg.PortableServer
+org.omg.PortableServer.CurrentPackage
+org.omg.PortableServer.POAManagerPackage
+org.omg.PortableServer.POAPackage
+org.omg.PortableServer.ServantLocatorPackage
+org.omg.PortableServer.portable
+org.omg.SendingContext
+org.omg.stub.java.rmi
+org.w3c.dom
+org.w3c.dom.bootstrap
+org.w3c.dom.events
+org.w3c.dom.ls
+org.xml.sax
+org.xml.sax.ext
+org.xml.sax.helpers
--- /dev/null
+++ b/tempfiles/groovyApi/package-list
@@ -0,0 +1,79 @@
+groovy.beans
+groovy.grape
+groovy.inspect
+groovy.inspect.swingui
+groovy.io
+groovy.jmx.builder
+groovy.json
+groovy.lang
+groovy.mock.interceptor
+groovy.model
+groovy.security
+groovy.servlet
+groovy.sql
+groovy.swing
+groovy.swing.binding
+groovy.swing.factory
+groovy.swing.impl
+groovy.text
+groovy.time
+groovy.transform
+groovy.ui
+groovy.ui.text
+groovy.ui.view
+groovy.util
+groovy.util.logging
+groovy.util.slurpersupport
+groovy.xml
+groovy.xml.dom
+groovy.xml.streamingmarkupsupport
+org.codehaus.groovy
+org.codehaus.groovy.ant
+org.codehaus.groovy.antlr
+org.codehaus.groovy.antlr.java
+org.codehaus.groovy.antlr.parser
+org.codehaus.groovy.antlr.treewalker
+org.codehaus.groovy.ast
+org.codehaus.groovy.ast.builder
+org.codehaus.groovy.ast.expr
+org.codehaus.groovy.ast.stmt
+org.codehaus.groovy.ast.tools
+org.codehaus.groovy.binding
+org.codehaus.groovy.bsf
+org.codehaus.groovy.classgen
+org.codehaus.groovy.classgen.asm
+org.codehaus.groovy.control
+org.codehaus.groovy.control.customizers
+org.codehaus.groovy.control.io
+org.codehaus.groovy.control.messages
+org.codehaus.groovy.groovydoc
+org.codehaus.groovy.jsr223
+org.codehaus.groovy.reflection
+org.codehaus.groovy.reflection.stdclasses
+org.codehaus.groovy.runtime
+org.codehaus.groovy.runtime.callsite
+org.codehaus.groovy.runtime.dgmimpl
+org.codehaus.groovy.runtime.dgmimpl.arrays
+org.codehaus.groovy.runtime.memoize
+org.codehaus.groovy.runtime.metaclass
+org.codehaus.groovy.runtime.powerassert
+org.codehaus.groovy.runtime.typehandling
+org.codehaus.groovy.runtime.wrappers
+org.codehaus.groovy.syntax
+org.codehaus.groovy.tools
+org.codehaus.groovy.tools.ast
+org.codehaus.groovy.tools.groovydoc
+org.codehaus.groovy.tools.groovydoc.gstringTemplates
+org.codehaus.groovy.tools.gse
+org.codehaus.groovy.tools.javac
+org.codehaus.groovy.tools.shell
+org.codehaus.groovy.tools.shell.commands
+org.codehaus.groovy.tools.shell.util
+org.codehaus.groovy.tools.xml
+org.codehaus.groovy.transform
+org.codehaus.groovy.util
+org.codehaus.groovy.vmplugin
+org.codehaus.groovy.vmplugin.v4
+org.codehaus.groovy.vmplugin.v5
+org.codehaus.groovy.vmplugin.v6
+org.codehaus.groovy.vmplugin.v7
